naive-http-caching/naive-http-caching-java-sbt/README.md

```diff
@@ -0,0 +1,35 @@
+# Header Manipulation and HTTP testing
```

Comment:
If the purpose of the recipe is to demonstrate header manipulation, I think the name of the directory/project should reflect that. If someone is looking for a generic header manipulation example and are not doing caching specifically, they might not think to look in something called "naive-http-caching".

Reply:
@TimMoore I think that the initial goal was not to demonstrate generic header manipulation, but indeed that is maybe a better option. 

Instead, we could have different, but very simple, ServiceCalls each demonstrating a different header manipulation use case. The If-None-Match and ETag would one of the possible use cases.

Reply:
I didn't know how to name the folders. The purpose of the recipe wasn't clear in my head when I started. :-(

Reply:
@renatocaval I like the idea. I'm only worried too many examples could make the sample too hard to follow. I think we'll find out if/when we add more calls.